File: 2_Kernel/JavaKernel/src/main/java/com/seagosoft/tws/data/CompletedOrderData.java
package com.seagosoft.tws.data;

import com.ib.client.Contract;
import com.ib.client.Order;
import com.ib.client.OrderState;

public class CompletedOrderData {

    public Contract contract;
    public Order order;
    public OrderState orderState;

    public CompletedOrderData(Contract contract, Order order, OrderState orderState) {
        this.contract = contract;
        this.order = order;
        this.orderState = orderState;
    }
}

File: 3_OrderSystem/KotlinEWrapperBackend/src/main/kotlin/com/seagosoft/tws/wrapper/structs/CompletedOrderData.kt
package com.seagosoft.tws.wrapper.structs

import com.ib.client.Contract
import com.ib.client.Order
import com.ib.client.OrderState


class CompletedOrderData(contract: Contract, order: Order, var orderState: OrderState) {
    var contract: Contract = contract
    var order: Order = order
}